### DISCUSSION TITLE
Add "Dependency Dashboard" showcase page?

### DISCUSSION THREAD

**Discussion by HonkingGoose** [renovatebot/renovate#11170] [Suggest an Idea]
A user [commented](
https://github.com/renovatebot/renovate/issues/11134#issuecomment-894242852) about our Dependency Dashboard:

>Awesome feature. It should have a better exposure in the documentation maybe with a dedicated section in getting started.

I also think the dashboard is a great feature, and I agree that we should promote it better, and increase the discoverability of the feature.

It would probably go well under the Getting Started section of the sidebar, with a new entry "Dependency Dashboard".

I could write sections on:

- Supported platforms
- How to enable the dashboard
- Use-cases:
  - Dependency Dashboard Approval workflow
  - Visibility into rejected/deferred updates
  - Other things that I'm not thinking of right now? Open for suggestions! 😉 
  
Let me know if this is something you want me to work on. And if you have ideas on how to make it better, please share!

**Comment by viceice** [comment]
I would suggest to enable it by default from v26.

We should add a note to the feature table that some platforms do miss this.

**Reply by HonkingGoose** [reply]
> I would suggest to enable it by default from v26.

I like this idea! We can be sure our users know about and see the dashboard, if we enable it by default for them! 😄

What happens if you turn on Dependency Dashboard on a platform that doesn't support it? Do you just not get a dashboard at all, or does it crash Renovate?

**Reply by viceice** [reply]
You simply get no dashboard. Everything else would be a bug 🐛

**Reply by HonkingGoose** [reply]
Okay, so there's no harm in turning it on the dashboard by default then... 😄 

If we decide to turn it on by default in `v26`, maybe we should add a new footer text to the dashboard:

```markdown
If you don't want this Dependency Dashboard, do .... to disable it.
```

This way people don't have to go searching in our docs for how to turn off the dashboard.
It might be a bit of a "surprise" for people who get automatically updated (github app users), so that's why I want to provide an easy way to learn how to turn it off again. 😄

**Reply by HonkingGoose** [reply]
@rarkins What do you think about turning on the Dependency Dashboard by default for all users in our `v26` release? This would be a BREAKING CHANGE in behavior.

**Reply by rarkins** [reply]
I don't consider this strictly to be a breaking change (it's very hard to define breaking when things come to user interface) but adding to v26 now is a good idea so there's no need to debate it. I prefer not to permanently pollute the dashboard with "how to turn it off" though, so maybe we just need to make sure that doing so is easily searchable in our docs? Especially near the top of the dependency dashboard page/section.

**Reply by HonkingGoose** [reply]
> I don't consider this strictly to be a breaking change (it's very hard to define breaking when things come to user interface) but adding to v26 now is a good idea so there's no need to debate it.

Can you maybe make the PR to enable the dashboard by default? I don't know where the toggle for this is in the code. 😄 

---

> I prefer not to permanently pollute the dashboard with "how to turn it off" though, so maybe we just need to make sure that doing so is easily searchable in our docs? Especially near the top of the dependency dashboard page/section.

Do you mean add a link to the top of the dashboard itself to the docs? Or do you mean a new page in our docs?

**Reply by rarkins** [reply]
I mean make sure that if someone searches for "dependency dashboard" then how to disable/enable it is right near the top. Or including if they search for "how to disable dependency dashboard" :)

BTW we should double check that repos with issues disabled don't cause us "noise" in the logs (e.g. no log warnings)

**Reply by rarkins** [reply]
![image](https://user-images.githubusercontent.com/6311784/128687476-1542ae72-be01-490a-8a13-083625e570da.png)

Maybe we could consider a standalone page describing dependency dashboard so that it's not only in the Configuration Options page?

**Reply by HonkingGoose** [reply]
> Maybe we could consider a standalone page describing dependency dashboard so that it's not only in the Configuration Options page?

That was my idea as well, having a separate page will make it easier to "stumble upon" this in the docs. And it might bump that page a bit in the search ranking...

My plan for the new page was:

> It would probably go well under the Getting Started section of the sidebar, with a new entry "Dependency Dashboard".
> 
> I could write sections on:
> 
>     * Supported platforms
> 
>     * How to enable the dashboard
> 
>     * Use-cases:
>       
>       * Dependency Dashboard Approval workflow
>       * Visibility into rejected/deferred updates
>       * Other things that I'm not thinking of right now? Open for suggestions! 😉
> 
> 
> Let me know if this is something you want me to work on. And if you have ideas on how to make it better, please share!

**Reply by rarkins** [reply]
Yes, I would love if you can work on this ASAP so that we're in a better position to enable the dashboard by default in v26, which I also want to release ASAP :)

**Reply by HonkingGoose** [reply]
Opened issue:

- https://github.com/renovatebot/renovate/issues/11177

to track:

> BTW we should double check that repos with issues disabled don't cause us "noise" in the logs (e.g. no log warnings)

And also opened issue:

- https://github.com/renovatebot/renovate/issues/11178

to track progress on the new docs page.
